purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesFactory.kt

```diff
@@ -44,7 +44,8 @@ internal class PurchasesFactory(
         configuration: PurchasesConfiguration,
         platformInfo: PlatformInfo,
         proxyURL: URL?,
-        overrideBillingAbstract: BillingAbstract? = null
+        overrideBillingAbstract: BillingAbstract? = null,
+        forceServerErrors: Boolean = false
```

Comment:
Note that there are 2 ways of setting the flag, when first initializing the SDK (from tests) or dynamically later by changing the flag in `AppConfig`. The reason for that is to be able to test situations with a fresh install where all requests fail initially. Otherwise, the initial requests performed during the SDK initialization won't be forced to be errors.

Reply:
That makes sense.